internal/api/v1/application/stage.go

```diff
@@ -396,44 +500,91 @@ func newPipelineRun(app stageParam) *v1beta1.PipelineRun {
 				"app.kubernetes.io/component":  "staging",
 			},
 		},
-		Spec: v1beta1.PipelineRunSpec{
-			ServiceAccountName: "staging-triggers-admin",
-			PipelineRef:        &v1beta1.PipelineRef{Name: "staging-pipeline"},
-			Params:             params,
-			Workspaces: []v1beta1.WorkspaceBinding{
-				{
-					Name:    "cache",
-					SubPath: "cache",
-					PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
-						ClaimName: app.MakePVCName(),
-						ReadOnly:  false,
+	}
+
+	job := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: app.Stage.ID,
+			Labels: map[string]string{
+				"app.kubernetes.io/name":       app.Name,
+				"app.kubernetes.io/part-of":    app.Namespace,
+				"app.kubernetes.io/created-by": app.Username,
```

Comment:
Is this the user that originally created the app? If yes, I think we need to grab the user that made the stage request and use that instead. "User A" may create the app but "User B" may stage it.

Reply:
According to
https://github.com/epinio/epinio/blob/933cbb467f849d94389b7ba2ad376850a1baafab/internal/api/v1/application/stage.go#L100
and
https://github.com/epinio/epinio/blob/933cbb467f849d94389b7ba2ad376850a1baafab/internal/api/v1/application/stage.go#L206
this is the user making the stage request.